cmd/spire-agent/cli/run/run.go

```diff
@@ -389,11 +406,27 @@ func parseTrustBundle(bundleBytes []byte, trustBundleContentType string) ([]*x50
 	return nil, fmt.Errorf("unknown trust bundle format: %s", trustBundleContentType)
 }
 
-func downloadTrustBundle(trustBundleURL string) ([]byte, error) {
+func downloadTrustBundle(trustBundleURL string, trustBundleUnixSocket string) ([]byte, error) {
+	var req *http.Request
+	client := &http.Client{}
```

Comment:
We may use the DefaultClient here (that's the one used by the http.Get function).
```suggestion
	client := http.DefaultClient
```